echopype/convert/set_groups_base.py

```diff
@@ -149,14 +149,14 @@ def set_nmea(self) -> xr.Dataset:
         ds = xr.Dataset(
             {
                 "NMEA_datagram": (
-                    ["time1"],
+                    ["nmea_time"],
                     raw_nmea,
                     {"long_name": "NMEA datagram"},
                 )
             },
             coords={
-                "time1": (
-                    ["time1"],
+                "nmea_time": (
+                    ["nmea_time"],
```

Comment:
How about `time_nmea` to be consistent with the noun-modifier pattern that's used elsewhere?

Reply:
Ugh just saw that there's `ping_time` that deviates from this pattern, and it would be odd to make it `time_ping`. Personally I feel it is cleaner for the other ones to be all like `time_X`...

Reply:
Oh huh I never thought about it that way. I personally see them both as a standalone/compound noun since 'ping' + 'time' refer to different things but together form this one distinct thing and 'ping time' could be operated on like "short + 'ping time'", and 'nmea time' I feel could be operated on similarly.